addons/api/fnc_setRadioAudioSource.sqf

```diff
@@ -0,0 +1,47 @@
+#include "script_component.hpp"
+/*
+ * Author: ACRE2Team
+ * Sets the audio source on the provided radio ID.
+ * Possible Values for ACRE_PRC148: ["INTAUDIO", "TOPAUDIO", "TOPSIDETON", "SIDEAUDIO", "SIDESIDETON"]
+ * Possible Values for ACRE_PRC152: ["INTAUDIO", "TOPAUDIO"]
+ * Possible Values for ACRE_SEM52SL: ["INTSPEAKER", "HEADSET"]
+ * Possible Values for ACRE_SEM70: ["INTSPEAKER", "HEADSET"]
+ *
+ * Arguments:
+ * 0: Radio ID <STRING>
+ * 1: Audio Source <STRING>
+ *
+ * Return Value:
+ * Success <BOOLEAN>
+ *
+ * Example:
+ * _success = ["ACRE_PRC148_ID_123", "INTAUDIO"] call acre_api_fnc_setRadioAudioSource;
+ *
+ * Public: Yes
+ */
+
+params [
+    ["_radioId", "", [""]],
+    ["_audioSource", "", [""]]
+];
+
+private _radioType = [_radioId] call acre_api_fnc_getBaseRadio;
+private _success = false;
+
```

Comment:
This would help to make the API easier to use by just letting users just pass a `true` to set speaker
but also adds clutter to the function and might not be needed

```
if (_audioSource isEqualTo true) then {
    _audioSource = switch (_radioType) do { 
        case "ACRE_PRC148";
        case "ACRE_PRC152": {"INTAUDIO"};
        case "ACRE_SEM70";
        case "ACRE_SEM52SL": {"INTSPEAKER"};
        default {"#"};
    };
};
```

Reply:
That should be a separate API function (wrapper of this one for example).

Reply:
I will add another function to enable/disable the speaker

Reply:
@Soldia1138 Added, please review.